src/components/network/network-panel.html

```diff
@@ -4,6 +4,13 @@
     <button class="btn btn-default glyphicon glyphicon-refresh btn-update"></button>
     <input class="form-control input-with-update">
   </div>
+  <div id="input-type" class="panel-section">
+    <div class="item-title">Input Type</div>
+      <div class="input-checkbox" id="gene-input-type">
+        <label class="radio-inline"><input type="radio" name="regex">regex</label>
```

Comment:
names for input:radio are required for the browser to identify that these radioboxes belong to the same group. We need to put the names on the parent <label> element.